### ISSUE TITLE
Serve command runs in background

### ISSUE THREAD

**Issue by beechnut** [18F/18f.gsa.gov#3819]
## Expected Behavior

When I run `./serve` and signal an interrupt (`ctrl+C`), the `serve` command should stop running and the site should stop working in the browser.

## Actual Behavior

When I run `./serve` and signal an interrupt (`ctrl+C`), `serve` continues running in the background and the local site continues to work.

## Steps to reproduce the behavior

- [ ] Follow local installation instructions and run `./serve`
- [ ] Visit the site (it should be running)
- [ ] Type `ctrl+C` to interrupt
- [ ] Refresh the tab with the site (it's still running, but shouldn't be)

## This issue is done when

- [ ] `serve` shuts down when given an interrupt signal (`ctrl+C`)

**Comment by beechnut** [comment]
[This line appears to be the culprit](https://github.com/18F/18f.gsa.gov/blob/b4348f97e5ab1b1eb3bff7708488f2003c4a38e4/serve#L10) — there's a `&` where I suspect we want a `&&`. But it's a little hard to tell, perhaps it's intentional that it's trying to run a couple processes in parallel.

**Comment by caleywoods** [comment]
I don't see this issue occurring on my machine. Sending `ctrl + c` to interrupt a running `./serve` causes the local site to stop responding when using `./serve` or `./serve-fast` and the PIDs associated with the running job/task disappear when doing a process list with `ps -a`. 

I'm not sure what the difference could be but some here's some info about my local setup:

* Intel chip macbook (non Apple Silicon/M1/M2/M3)
* I'm using [Kitty](https://sw.kovidgoyal.net/kitty/) for a terminal app
* I'm using zshell and not bash

I tried a few different things to see if it changed the behavior I see:

* I updated the `./serve` script to run in a bash shell by changing `#!/bin/sh` at the top of the script to `#!/bin/bash`. Running this script in Kitty and in the default terminal application made no difference.
* I tried running the `./serve` script in the default terminal application and that made no difference.
* I ran `chsh -s /bin/bash` to temporarily switch over to a bash shell and then running an unmodified `./serve` script resulted in no changes, the process stops as expected when sent an interrupt.
